live-examples/html-examples/text-content/figcaption.html

```diff
@@ -0,0 +1,4 @@
+<figure>
+    <img src="/media/examples/hamster.jpg" alt="hamster">
```

Comment:
In these examples we prefer to close tags even when it's not needed:

`<img src="/media/examples/hamster.jpg" alt="hamster" />`